### ISSUE TITLE
BA2/Appliance GUI Xsession Errors

### ISSUE THREAD

**Issue by AaronJowitt** [AaronJowitt/Rivendell#1]
remember how every week or our BA2 server fills up a 96GB partition with a massive .xsession-error (or similar) file?  You were looking at an extract of the file a few months ago, and said it was a GUI related error.  Did you find what was causing it - as the server still comes to a grinding halt when that partition fills up with that same error file.

**Comment by ElvishArtisan** [comment]
Has the name on this system been changed by any chance?  Was the system rebooted after making a name change?

**Comment by AaronJowitt** [comment]
Yes - changed the name.  Not sure it we rebooted immediately after that (it _probably_ was), but it has since been rebooted many times.
